e2e/integration/frontity-01/script.spec.js

```diff
@@ -20,7 +20,7 @@ describe("Script", () => {
       .its("moment")
       .should("exist")
       .then((moment) => {
-        expect(moment()._isAMomentObject).to.equal(true);
+        expect(moment()._isAMomentObject).toBe(true);
```

Comment:
🤔  huh, wondering why do you think we should use Jest's expect? `cypress` has the chai one already 🤷‍♂️

Reply:
Wondering the same!

Just my two cents, @luisherranz: https://github.com/frontity/frontity/pull/672#discussion_r563604147

Reply:
It was simply a personal preference, to lean the commands on only one `expect` instead of two.

Reply:
Gotcha!